package.json

```diff
@@ -22,18 +22,14 @@
     "mustache"
   ],
   "dependencies": {
+    "can": "^6.4.0",
     "loader-utils": "^1.1.0"
   },
   "devDependencies": {
-    "can-component": "^4.2.2",
-    "can-stache": "^4.0.0",
-    "can-stache-ast": "^1.0.0",
```

Comment:
Is `can-stache-ast` not needed? It is used by the index.js file. I think it should be a dependency.